daemon/SecretsImpl/secrets.cpp

```diff
@@ -702,14 +828,15 @@ bool Daemon::ApiImpl::SecretsRequestQueue::compareTestCipherText(
     if (previousData.isEmpty()) {
         if (writeIfNotExists) {
             // first time, write the file.
-            s = dataProtector.putData(testCipherText);
+            s = dataProtector.putData(cipherPluginName.toUtf8() + '\n' + testCipherText);
             return s == DataProtector::Success;
         } else {
             qCWarning(lcSailfishSecretsDaemon) << "Unable to read ciphertext data from nonexistent lock code check file";
             return false;
         }
     } else {
-        if (previousData != testCipherText) {
+        const QByteArray previousDataExceptPluginName = previousData.mid(previousData.indexOf('\n')+1);
```

Comment:
Bad tabulation of `)+1`. Should be `) + 1`.

Reply:
done